w06-ruby/w06-d1-homework/README.md

```diff
@@ -0,0 +1,179 @@
+# Ruby Koans
+
+Go to the Ruby Koans [website](http://rubykoans.com/) and click the link to download the koans. Open the zip file and save it in your `dev` directory. `cd` into the `koans` directory and run `ls`. Wow, that's a lot of files!
+
+Each file is a self-contained exercise that will teach you a bit about Ruby, using test driven development. Essentially you will be given code that has something wrong with it, as well as instructions about what needs to be fixed. Then you make the necessary changes for the tests to pass.
+
+For example, running this file in the Terminal
+```bash
+:> ruby about_hashes.rb
+```
+will output this:
+```ruby
+AboutHashes#test_creating_hashes has damaged your karma.
+
+The Master says:
+  You have not yet reached enlightenment.
+
+The answers you seek...
+  Expected "FILL ME IN" to equal Hash
+
+Please meditate on the following code:
+  about_hashes.rb:6:in `test_creating_hashes'
+
+mountains are merely mountains
+your path thus far [X_________________________________________________] 0/12
+```
+
+While it may look verbose, all this output is doing is explaining why the code isn't running and stating which line (and which method, if applicable) to fix. Each time fix that is made will provide a new prompt when the file is run again.
```

Comment:
"Each time fix that is made"  <-- not sure about this, maybe "each time a fix is made X will provide"